concepts.wip/if-expressions/about.md

```diff
@@ -1,52 +1,67 @@
 # About
 
-`if`-expressions in Julia are similar to those seen in other languages:
+In Julia, like many other languages, allow for the following with `if`-expressions:
+
+- Chained if-expressions
+- Nested if-expressions
+
+~~~~exercism/note
+`if`-expressions as defined here will encompass the following syntaxes:`if`, `if-else` and `if-elseif-else` statements.
```

Comment:
```suggestion
`if`-expressions as defined here will encompass the following syntaxes:`if`, `if-else` and `if-elseif-else` statements.
```

This alludes to other syntaxes existing, could you perhaps add a link to their concept pages? I'm guessing the conditionals page?

Reply:
Done.